extra_tests/snippets/stdlib_random.py

```diff
@@ -23,6 +23,15 @@
 sampled = [0, 2, 1]
 assert random.sample(list(range(3)), 3) == sampled
 
+# random.randbytes
+random.seed(1234)
+zero_size_buf = random.randbytes(0)
+assert type(zero_size_buf) is bytes
+assert len(zero_size_buf) == 0
+non_zero_buf = random.randbytes(4)
+assert type(non_zero_buf) is bytes
+assert list(non_zero_buf) == list(b'\xb9\x7fi\xf7')
```

Comment:
Try experimenting with `testutils.skip_if_unsupported` (defined [here](https://github.com/RustPython/RustPython/blob/5254265e3ad2164e47430b7137446fc9309cb210/extra_tests/snippets/testutils.py#L74-L83)). [Here's an example of it in action.](https://github.com/RustPython/RustPython/blob/5254265e3ad2164e47430b7137446fc9309cb210/extra_tests/snippets/stdlib_math.py#L26-L35)